packages/core/addon/config.ts

```diff
@@ -29,6 +29,7 @@ export type SvApi = {
 export type Addon<Args extends OptionDefinition> = {
 	id: string;
 	alias?: string;
+	shortDescription: string;
```

Comment:
should this be optional like the others? especially if we don't want community adders to have to fill them out?
```suggestion
	shortDescription?: string;
```